### PR TITLE
retrieve all users from client_user table

### LINKED ISSUE #13: 10. Retrieve  list  all User
Build a controller that retrieves all the users

### PR DESCRIPTION
**Pull request by nancytaen**
Created a route to retrieve all users from client_user table
* tested using localhost:3000/customers router

### CODE REVIEW

File: Backend/app/models/customer.model.js [hunk 1]
```diff
@@ -2,10 +2,24 @@ const sql = require("./db.js");
...
-  this.active = customer.active;
+  this.email = customer.email;
+  this.phone = customer.phone;
 };
 
 
+Customer.retrieveAll = function() {
+  return new Promise(function(resolve, reject){
+    sql.query('SELECT * FROM client_users', function(err, rows) {
+      if (err) throw err;
```
Comment (youssefsoli) [inline Backend/app/models/customer.model.js]: Use reject() for the error instead of throwing.

Comment (nancytaen) [reply Backend/app/models/customer.model.js]: Good point, thanks!

**Comment by nancytaen** [comment]
I will make the pr to sprint1 branch

**Comment by dragonite20** [comment]
LGTM! Good JOB!

**Pull request merged**